wiki/Developer-Documentation.md

```diff
@@ -325,74 +174,17 @@ To execute,
 ------------------------------------
 Python applications can access QFS by using the python extension module. This
-section describes how to build and install the python extension module.
-
-### Building Python Extension Module
-
-One can build the python module after building QFS client library.
-
-1. With build directory as ~/code/qfs/build, build and install the QFS client
-library as described above.
-
-        $ cd ~/code/qfs/build/release
-        $ cmake ~/code/qfs/ && make && make install
-
-
-1. From `~/code/qfs/build/release`, run the python setup script. This will
-create the python extension module under
-`~/code/qfs/build/release/build/<distdir>/`.
-
-        $ cd ~/code/qfs/build/release
-        $ python ~/code/qfs/src/cc/access/kfs_setup.py build
-
-The file `~/code/qfs/build/build/<distdir>/qfs.so` will be created at the end of
-last step.
-
-**Note:** If you want to generate the modules in a path different from 'build',
-issue the following command:
-
-    $ python ~/code/qfs/src/cc/access/kfs_setup.py build --build-base ./pybuild
-
-### Installing The Python Module
-The `qfs.so` library needs to be installed in the site-packages for python
-applications to use it. Default installation may require write access to
-privileged directories. Custom installation allows installing at an alternate
-location.
-
-- Follow these steps for default installation.
-
-        $ cd ~/code/qfs/build/release
-        $ python ~/code/qfs/src/cc/access/kfs_setup.py install
-
-If `/usr/lib64/python2.6/site-packages/` is where the site packages go then
-`qfs.so` will go here.
-
-- Follow these steps for custom installation.
-
-        $ cd ~/code/qfs/build/release
-        $ python ~/code/qfs/src/cc/access/kfs_setup.py install --home ./qfs_python
-
-This command will install the python module at
-`~/code/qfs/build/release/qfs_python/lib[64]/python/qfs.so`
-
-**Note:** To allow python run-time environment to find this package, one will
-need to add the package path to `PYTHONPATH`. The package path is the path
-leading to `qfs.so`. In the above example, one would do:
-
-    $ export PYTHONPATH=${PYTHONPATH}:~/code/qfs/build/release/qfs_python/lib[64]/python
-
+section describes how to build and install the python extension module. To build
+the python module, use the command `make python`. The file
+`build/build/<distdir>/qfs.so` will be created at the end.
 
 ### Developing a Python Client
 Python applications use the python QFS extension module, `qfs.so`, to get at the
 C++ QFS client library. The example program
-`~/code/qfs/examples/python/qfssample.py` illustrates how to write a Python
-client for QFS.
+`examples/python/qfssample.py` illustrates how to write a Python client for QFS.
 
 Set `$PYTHONPATH` accordingly if you are using a `qfs.so` install path different
 from the default path, so that the python run-time can detect the `qfs.so`.
-Also, ensure that LD_LIBRARY_PATH has the path to the QFS libraries.
```

Comment:
nit: I agree that these commands are trivial for most developers, but commands are easier for human eye to parse at first sight than reading full sentences. Personally, in any documentation, I first seek for commands and then I read the text around them if a command seems to be something that I need.